circle.yml

```diff
@@ -2,7 +2,7 @@ machine:
   environment:
     DOCKER_IMAGE_NAME: prom/node-exporter
     QUAY_IMAGE_NAME: quay.io/prometheus/node-exporter
-    DOCKER_TEST_IMAGE_NAME: quay.io/prometheus/golang-builder:1.6-base
+    DOCKER_TEST_IMAGE_NAME: quay.io/prometheus/golang-builder:1.7-base
```

Comment:
@sdurrheimer It looks like the quay image for golang-builder is 1.7.3, I'm not sure what the difference is between `-base` and `-main`.